File: 8-kyu/how-much-water-do-i-need-/python/solution.py
def how_much_water(water, load, clothes):
    if load * 2 < clothes:
        return 'Too much clothes'
    elif clothes < load:
        return 'Not enough clothes'
    calculated_water = water * 1.1 ** abs(load - clothes)
    return round(calculated_water, 2)

File: 8-kyu/how-much-water-do-i-need-/javascript/solution.js
const howMuchWater = (water, load, clothes) => {
  if (load * 2 < clothes) {
    return "Too much clothes";
  }
  if (clothes < load) {
    return "Not enough clothes";
  }
  const neededWater = water * 1.1 ** Math.abs(load - clothes);
  return parseFloat(neededWater.toFixed(2));
};
